test/controllers/api/v1/ping_thing_controller_test.rb

```diff
@@ -43,4 +43,45 @@ def setup
 
     assert_equal expected_response, response_to_json(@response.body)
   end
+
+  test "create_multiple request without token returns error" do
+    post api_v1_ping_thing_batch_path(network: "testnet")
+    assert_response 401
+    expected_response = { "error" => "Unauthorized" }
+
+    assert_equal expected_response, response_to_json(@response.body)
+  end
+
+  test "create_multiple request with token should return 200 and save the data" do
+    params_batch = {
+      transactions: [
+        @params_sample, @params_sample
+      ]
+    }
+
+    post api_v1_ping_thing_batch_path(
+      network: "testnet"
+    ), headers: { "Token" => @user.api_token }, params: params_batch
```

Comment:
```suggestion
    ), headers: { "Token" => @user.api_token }, params: params_batch.to_json
```
I think it would be more appropriate to send params in json format